validate.lic

```diff
@@ -399,13 +399,13 @@ class DRYamlValidator
   end
 
   def assert_that_barb_buffs_are_in_base_spells(settings)
-    return less setting.buff_nonspells['barb_buffs']
+    return unless settings.buff_nonspells['barb_buffs']
 
     supported_buffs = get_data('spells').barb_abilities
 
-    setting.buff_nonspells['barb_buffs']
-        .select { |name| supported_buffs.value?(name) }
-        .each { |desc| error("Barb ability '#{name}' not yet supported. Please make an issue on Git for it to be added!") }
+    settings.buff_nonspells['barb_buffs']
+        .select { |name| !supported_buffs.include?(name) }
```

Comment:
My new branch actually has this in their style guide, so it's best I get used to it now.